Interface/Harp.Olfactometer/Harp.Olfactometer.csproj

```
@@ -17,7 +17,7 @@
     <PackageLicenseFile>LICENSE</PackageLicenseFile>
     <PackageOutputPath>..\bin\$(Configuration)</PackageOutputPath>
     <TargetFrameworks>net462;netstandard2.0</TargetFrameworks>
-    <VersionPrefix>0.1.0</VersionPrefix>
+    <VersionPrefix>0.1.1</VersionPrefix>
```

```suggestion
    <VersionPrefix>0.2.0</VersionPrefix>
```

Feels like this should be a bump to the minor version since new registers and features were added, no?

---

The rational was based on @bruno-f-cruz comment: " I think that prior to 1.x, we tend to use the patch from semver to indicate backward compatible. If these are indeed fully backwards compatible i would tag this release as 0.1.1."

---

Sounds great, in that case @artursilva0 changing this version to 0.2.0 and the name of the calibration register are the only two changes needed, everything else LGTM!